spec/results-model-spec.js

```diff
@@ -33,6 +35,10 @@ describe("ResultsModel", () => {
       expect(resultsModel.getPathCount()).toBe(1);
       expect(resultsModel.getMatchCount()).toBe(6);
       expect(resultsModel.getPaths()).toEqual([editor.getPath()]);
+      if (parseFloat(atom.getVersion()) >= 1.17) {
```

Comment:
Please see the first reference. The same condition has been added to the code of this package. Since the package doesn't specify a minimum version I think it should be changed.